benchmarks/scripts/benchmark_clifford_circuit.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# ==============================================================================

```

Comment:
Would you be able to revert the files where there are little to no changes made to the actual code so that the diff view is easier to read ?

Reply:
Sure, let me update the PR soon :) thank you Michael.